src/applications/proxy-rewrite/proxy-rewrite-whitelist.json

```diff
@@ -18,7 +18,7 @@
     {
         "hostname": "www.oprm.va.gov",
         "pathnameBeginning": "/",
-        "cookieOnly": true 
+        "cookieOnly": false 
```

Comment:
the teamsite header allows for production deployment behind a flag that's controlled by a cookie. setting this to false removes the cookie check and effectively launches the va.gov header to production oprm.va.gov